frontend/components/common/BaseValueInput.vue

```diff
@@ -3,14 +3,17 @@
         <Tooltip :visible="!!errorMessage" placement="topLeft" :title="errorMessage">
             <Input
                 v-model="inputText"
-                class="Input"
-                :class="{ Error: !!errorMessage }"
+                class="Input pr-8"
+                :class="{ Error: !!errorMessage, 'pr-10': auctionType === 'surplus' }"
                 :disabled="disabled"
                 @focus="hideMaxValue"
                 @blur="showMaxValueIfEmpty"
             />
             <div class="Overlay right-2" :class="{ 'opacity-50': disabled }">
-                <format-currency v-if="!inputValue && maxValue" :value="maxValue" />&nbsp;DAI
+                <div v-if="auctionType === 'surplus'">
+                    <format-currency v-if="!inputValue && minValue" :value="minValue" /> MKR
+                </div>
+                <div v-else><format-currency v-if="!inputValue && maxValue" :value="maxValue" />&nbsp;DAI</div>
```

Comment:
Can't you just provide `currency` prop instead?

Reply:
The problem is, that we have the space between the cursor in the input field and the currency title on the right hardcoded.

![Screenshot 2022-07-14 at 09 58 59](https://user-images.githubusercontent.com/30908158/178932341-cb6ce4cb-0412-42e1-ac70-9bf2a3614096.png)

This means that if we add a new currency and the symbol is slightly longer or shorter, the cursor spacing will not adjust and look wonky. 

I do think we should look into rewriting or adjusting the component in the future, however I thought as we are a little bit pressed for time, this does not need to be done in this PR.

Reply:
> the space between the cursor in the input field and the currency title on the right hardcoded.

It's seems to be super easy to instead of adding the hardcoded padding to add the `invisible` div with the currency text. That wouldn't make a long time, but is not "hacky"

Reply:
We position the currency symbol with absolute. As far as I know I cannot insert a element into the AntD Input field, so I am not sure what you mean here. Could you give me an example? 😊

Reply:
There is even a slot called `suffix`
https://2x.antdv.com/components/input#components-input-demo-presuffix

Reply:
![Screenshot 2022-07-14 at 11 36 57](https://user-images.githubusercontent.com/30908158/178952510-85d3fbf9-4ce9-4c2f-a0d8-db61c0f2367c.png)

Sadly, their `suffix` does not seem to apply the correct padding. 

If I switch the version on their website to V1 (the one we are using, you linked to the V2 docs), you can see that even they have the same padding issues in their demo.
![Screenshot 2022-07-14 at 11 39 34](https://user-images.githubusercontent.com/30908158/178953009-ab3b59a1-19fd-438d-a132-24a2714d7e9f.png)

Reply:
Ok, I give up, that's the issue for another time then